product/reports/120_Configuration Management - Providers/050_Watermark Vms per Provider.yaml

```diff
@@ -31,9 +31,9 @@ sortby:
 - resource_name
 headers:
 - Activity Sample - Month (YYYY/MM)
+- Region
 - Asset Name
 - VM Count Total (Max)
-- Region
```

Comment:
I'm good with this, headers order needs to match col_order, which they now do. 👍